src/core/Tile/Tile.test.tsx

```diff
@@ -221,3 +221,8 @@ it('should propagate misc props correctly', () => {
   expect(child).toBeTruthy();
   expect(child.textContent).toBe('test-content');
 });
+
+it('should render actionable tile', () => {
```

Comment:
Should we write a test for tabbing to an actionable tile?

Reply:
And what would you like me to test there?

Reply:
I was thinking appending this test with this

`expect(actionTile.getAttribute('tabIndex')).toBe('0');`

I guess I don't see other components have a test for this, so I guess we're good here